modules/rosa-sts-creating-cluster.adoc

```diff
@@ -22,7 +22,7 @@ When creating your cluster, use the following default options to configure your
 
 . You can create a cluster by specifying the specifying the custom settings shown below or by using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
 +
-Creating a cluster can take up to 40 minutes.
+Creating a cluster can take up to 40 minutes. During this waiting period, you can configure identity providers that determine how users log in to access the cluster. For more information, see Next Steps below.
```

Comment:
The below steps are for how to configure oidc-providers and how to create operator-roles on AWS. Those steps are not used for user to configure `you can configure identity providers that determine how users log in to access the cluster` . I think this sentence should be moved to the last step 5 in file https://deploy-preview-34534--osdocs.netlify.app/openshift-rosa/latest/rosa_getting_started_sts/rosa-sts-creating-cluster.html#rosa-sts-creating-cluster_rosa-sts-getting-started
And
We should better to clarify that cluster will keep pending until the oidc-provider created in step 4